ci/ansible/roles/deploy/tasks/common_setup.yml

```diff
@@ -47,7 +48,26 @@
     fstype: nfs
     boot: true
     path: "{{ ipinfo_path }}"
+  when: local_dev is not defined or not local_dev
 
+- name: Fetch sample IPinfo location database
+  ansible.builtin.get_url:
+    url: https://github.com/ipinfo/sample-database/raw/main/IP%20Geolocation/ip_geolocation_sample.mmdb
+    dest: /var/ipinfo/standard_location.mmdb
+  when: local_dev is not defined or not local_dev
+
+- name: Fetch free continent database
+  ansible.builtin.get_url:
+    url: https://country.io/continent.json
+    dest: /var/ipinfo/continent.json
+  when: local_dev is not defined or not local_dev
+
+- name: Fetch sample IPinfo ASN database
+  ansible.builtin.get_url:
+    url: https://github.com/ipinfo/sample-database/raw/main/ASN%20Database/asn_sample.mmdb
+    dest: /var/ipinfo/asn.mmdb
+  when: local_dev is not defined or not local_dev
+  
```

Comment:
For clarity it might be beneficial to move the URL's (and destinations?) to the default env vars